vulcan/config.yml.template

```diff
@@ -28,6 +28,9 @@ default: &default
   :token_name: JANUS_DEV_TOKEN
   :log_level: info
   :archimedes_run_image: etnaagent/archimedes
+  :archimedes_interpreters:
```

Comment:
Will this get translated to production `config.yml`?

Reply:
Nope, we'll want to add this into production.

```
./bin/portainer-login
./bin/portainer-edit-config vulcan.production.yml
```

Reply:
Those are some nice / handy scripts for updating the config and deploying! :clap: :tada: